### ISSUE TITLE
Content inset is incorrect in iOS device

### ISSUE THREAD

**Issue by HaloWang** [react-native-community/react-native-webview#559]
**Bug description:**

Content inset is incorrect in iPhone.

**To Reproduce:**

I upload this [repo](https://github.com/HaloWang/rnwebviewbug)

**Expected behavior:**

React-Native and Native have same behavior.

**Screenshots/Videos:**

**Environment:**
```
React Native Environment Info:
System:
  OS: macOS 10.14.4
  CPU: (8) x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  Memory: 244.08 MB / 16.00 GB
  Shell: 5.3 - /bin/zsh
Binaries:
  Node: 11.14.0 - /usr/local/bin/node
  Yarn: 1.15.2 - /usr/local/bin/yarn
  npm: 6.9.0 - /usr/local/bin/npm
  Watchman: 4.9.0 - /usr/local/bin/watchman
SDKs:
  iOS SDK:
    Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
  Android SDK:
    API Levels: 19, 23, 25, 26, 27, 28
    Build Tools: 23.0.1, 23.0.3, 25.0.2, 26.0.2, 27.0.3, 28.0.3
    System Images: android-23 | Intel x86 Atom, android-23 | Intel x86 Atom_64, android-23 | Google APIs ARM EABI v7a, android-23 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom_64, android-27 | Google Play Intel x86 Atom
IDEs:
  Android Studio: 3.3 AI-182.5107.16.33.5264788
  Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
npmPackages:
  react: 16.8.3 => 16.8.3 
  react-native: 0.59.5 => 0.59.5 
npmGlobalPackages:
  react-native-cli: 2.0.1
  react-native-create-library: 3.1.2
  react-native-git-upgrade: 0.2.7
```

```
"react-native-webview": "^5.8.1"
```

**Comment by ptrocki** [comment]
Probably the same related issues:
https://github.com/facebook/react-native/issues/23988
https://github.com/facebook/react-native/issues/23480

I checked code, and probably scrollview does not update contentInset properly.

**Comment by ptrocki** [comment]
Changing import to `import { WebView } from 'react-native';` it works correctly for version:
```
react: 16.8.3 => 16.8.3 
react-native: 0.59.8 => 0.59.8 
```

**Comment by HaloWang** [comment]
`<WebView useWebKit={false} ...` is not working for me.

**Comment by Titozzz** [comment]
fixed by #603
